src/UCTNode.h

```diff
@@ -110,7 +115,7 @@ class UCTNode {
     // UCT eval
     float m_policy;
     // Original net eval for this node (not children).
-    float m_net_eval{0.0f};
+    float m_net_eval{2.0f};
```

Comment:
Totally unclear why this is changed.

Reply:
It's changed to a value to indicate uninitialized status because I tried to use fpu instead when net_eval is invalid: https://github.com/gcp/leela-zero/pull/1772/commits/5976a2f996caf5d33723e56f517c05634c917563#diff-28216771f77a09f4e0782ad5404fdd28R326  https://github.com/gcp/leela-zero/pull/1772/commits/5976a2f996caf5d33723e56f517c05634c917563#diff-28216771f77a09f4e0782ad5404fdd28R89
However that idea is abandoned and the change is unnecessary now.